frontend/public/static/locale/es/translate.ftl

```diff
@@ -9,10 +9,10 @@
 ## Editor Menu
 ## Allows contributors to modify or propose a translation
 
-editor-EditorMenu--button-copy = Copiar
-editor-EditorMenu--button-clear = Borrar
-editor-EditorMenu--button-save = Guardar
-editor-EditorMenu--button-suggest = Sugerir
+editor-EditorMenu--button-copy = COPIAR 
```

Comment:
Mind the extra space.

Reply:
Did you want me to remove the space so there are no spaces or just one space?

Reply:
There's an extra ("COPIAR ") which should be removed ("COPIAR").